src/NuGet/Microsoft.Net.Compilers.Toolset/Microsoft.Net.Compilers.Toolset.Package.csproj

```diff
@@ -0,0 +1,59 @@
+﻿<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
```

Comment:
@dsplaisted, @nguerrera can you all take a look  at this file? It's the multi-TFM NuGet tool project. It functions on my machine but I'm sure I've done unnecessarily evil things here.